packages/config/src/Config.ts

```diff
@@ -501,6 +501,14 @@ export function readConfigJson(
     exp.version = pkg.version;
   }
 
+  if (exp && !exp.sdkVersion) {
+    try {
+      const packageJsonPath = resolveModule('expo/package.json', projectRoot, exp);
+      const expoPackageJson = require(packageJsonPath);
+      exp.sdkVersion = expoPackageJson.version;
+    } catch (_) {}
```

Comment:
Maybe we can throw an error here, because that allows us to guarantee that the configuration object returned by this function *always* has `sdkVersion`. IMHO this is better than failing in a weird way later.

We can make the error describe how to fix this, i.e. run npm/Yarn to ensure `expo` is installed in `node_modules`.